lib/MongoDB/Collection.pm

```diff
@@ -1274,11 +1280,12 @@ collection.
 =cut
 
 sub rename {
-    my ( $self, $new_name ) = @_;
+    my ( $self, $new_name, $options ) = @_;
```

Comment:
Adding a new options hash here is a good call.  Per comments elsewhere, we should pull out and validate a session explicitly and then in the Op only pass an explicit `session => $session` rather than the entire options hash.